scss/standalone/patterns_grid-breakout.scss

```diff
@@ -0,0 +1,10 @@
+@import '../base';
+@include vf-base;
+
+@import '../patterns_grid';
+@include vf-p-grid;
+@include vf-l-fluid-breakout;
```

Comment:
You need to `@import '../layouts_fluid-breakout';` because it's moved to separate file now.

Reply:
done